OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -40,6 +40,9 @@ public abstract class SupportPowerInfo : UpgradableTraitInfo
 
 		public readonly bool DisplayTimer = false;
 
+		[Desc("Defines to which players the timer is shown when 'DisplayTimer' is true.")]
+		public readonly Stance DisplayTimerStances = Stance.Ally | Stance.Neutral | Stance.Enemy;
```

Comment:
This makes `DisplayTimer` redundant, so please remove that field and default this to none.